catalog/ui/search-ui/standard/src/main/webapp/css/styles.css

```diff
@@ -1 +1 @@
ht:12px}.margin-top-3{margin-top:3px}.margin-top-6{margin-top:6px}.margin-top-12{margin-top:12px}.margin-bottom-3{margin-bottom:3px}.margin-bottom-6{margin-bottom:6px}.margin-bottom-12{margin-bottom:12px}.indent-3{text-indent:6px}.indent-6{text-indent:6px}.indent-12{text-indent:12px}.indent-18{text-indent:18px}.indent-24{text-indent:24px}.horz-padding-6{margin-right:6px;margin-left:6px}.horz-padding-3{margin-right:3px;margin-left:3px}.horz-padding-12{margin-right:12px;margin-left:12px}.filter-view .facet-selector-toggle{margin-top:-3px}.filter-view i{cursor:pointer;color:#008ab8}.filter-view i:hover{color:#5c5c5c}.filter-view i:active{color:#7ab800}.filter-view .filter-status{border-top:1px solid #444;text-align:center;cursor:pointer}.filter-view .filter-menu{display:none}.filter-view h6{text-transform:capitalize}.filter-view h6 i{font-size:19px}.filter-view .filter-table td select,.filter-view .filter-table td input{width:auto;border-radius:4px;font-size:12px;height:28px;margin-bottom:0}.filter-view .filter-table td select{padding:0}.filter-view .filter-table td .value-input-group,.filter-view .filter-table td .operator-group,.filter-view .filter-table td .metadata-field-group,.filter-view .filter-table td .button-group,.filter-view .filter-table td .geo-button-group{float:left;vertical-align:top}.filter-view .filter-table td .geo-button-group{padding:4px 6px 0 6px}.filter-view .filter-table td .metadata-field-group{margin-right:5px}.filter-view .filter-table td .metadata-field-group select{width:80px;margin-bottom:0}.filter-view .filter-table td .operator-group{margin-right:5px}.filter-view .filter-table td .operator-group select{width:80px;margin-bottom:0}.filter-view .filter-table td .value-input-group input{width:180px;margin-bottom:0}.filter-view .filter-table td .value-input-group input.geo-field{width:159px}.filter-view .filter-table td .button-group{float:right;padding-top:4px;padding-right:6px;padding-left:6px;cursor:pointer}.filter-view .field-value{font-size:small}.filter-view.active .filter-menu{display:block}@media (min-width:700px){.full-screen-search .filter-view .metadata-field-group select{width:auto}.full-screen-search .filter-view .operator-group select{width:auto}.full-screen-search .filter-view .value-input-group input{width:400px;margin-bottom:0}}@media (min-width:1000px){.full-screen-search .filter-menu{width:1000px;margin:0 auto}}select.ddf,.ddf.select{color:#fff;background:none #555;border:1px solid #666}select.ddf option,.ddf.select option{color:#fff!important;background-color:#555!important}table.table.vertical-shrink-rows td,table.table.vertical-shrink-rows th{padding-top:3px;padding-bottom:3px}table.table.no-horizontal-padding td,table.table.no-horizontal-padding th{padding-left:0;padding-right:0}table.table.no-bottom-margin{margin-bottom:0}table.table.attached{margin-top:-21px}table.table td.middle,table.table th.middle{vertical-align:middle}input[type=checkbox],input[type=radio]{cursor:pointer}.twipsy{display:none}
```

Comment:
This file should not be source-controlled. Delete it (and any others) and fix the git-ignore please. ;) https://github.com/codice/ddf/blob/master/.gitignore#L15